packages/react-interactive-tab/package.json

```diff
@@ -2,13 +2,14 @@
   "name": "@fluentui-contrib/react-interactive-tab",
   "version": "0.0.1",
   "dependencies": {
+    "@fluentui/react-jsx-runtime": ">=9.0.29 < 10.0.0",
```

Comment:
based on convo in previous thread

```suggestion
    "@fluentui/react-jsx-runtime": "^9.0.29",
```

Reply:
The `check-dependencies` task fails on CI, we might need to update other packages to this format in one go. I'd prefer to do this in scope of other PR, not related to the specific pacakage.

![image](https://github.com/user-attachments/assets/be8a3c36-8352-4f4f-9df3-c42eaff96b43)

Reply:
yeah that sounds good, in order to pass pipeline you'll need to ignore your package deps from syncpack